### DISCUSSION TITLE
[nuxt 3] redirect()

### DISCUSSION THREAD

**Discussion by paulvonber** [nuxt/nuxt#17293] [Questions]
Is there anything similar or the same as redirect() function like in nuxt2?

**Comment by ghost** [comment]
I would also like to know

**Comment by wobsoriano** [comment]
https://github.com/nuxt/framework/discussions/2004

https://github.com/nuxt/framework/discussions/906

**Comment by psycongaroo** [comment]
bump
any news about that?

I need to trigger 404 if useFatch fails.

**Reply by Bjornnyborg** [reply]
@shinguweb96 for triggering a 404, see my [comment here](https://github.com/nuxt/framework/discussions/2004#discussioncomment-2227374): Still looking for the redirect solution though. 😊

**Comment by Lanayx** [comment]
There is a `navigateTo` function available ([docs](https://v3.nuxtjs.org/docs/directory-structure/middleware))

**Comment by justin-schroeder** [comment]
For the most part you should use `navigateTo`, but currently it doesn't support external redirects, at least until #5022 is merged. Here's a little composable you can use to perform external redirects too: https://gist.github.com/justin-schroeder/f583797c702155f32c9078d1add2a594
